ProcessLib/LiquidFlow/LiquidFlowLocalAssembler-impl.h

```diff
@@ -104,6 +105,10 @@ void LiquidFlowLocalAssembler<ShapeFunction, IntegrationMethod, GlobalDim>::
         const double mu =
             _material_properties.getViscosity(p, _reference_temperature);
 
+        pos.setIntegrationPoint(ip);
+        auto const& permeability = _material_properties.getPermeability(
+            material_id, t, pos, _element.getDimension(), p, 0.0);
```

Comment:
```suggestion
            material_id, t, pos, _element.getDimension(), p, _reference_temperature);
```
It's used above in `mu` computation.